Monika After Story/game/script-topics.rpy

```diff
@@ -6288,6 +6288,115 @@ label monika_yellowwp:
             m 1a "It'll definitely be an interesting read for you."
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_driving",category=['monika'],prompt="Can you drive?",random=True))
+
+label monika_driving:
+    m 1d "Hm? Can I drive?"
+    m 1c "I never really thought about getting a driver's licence."
+    m 3a "Public transportation is enough for me usually."
+    m "Walking can be really nice sometimes too!"
+    m "I'm not sure I would even have had time to learn to drive with school and all the activities I did."
+    m "Not to mention that I'm trying to reduce my carbon footprint anyway."
+    m 1a "I guess you could say I never really had the need to be able to drive."
+    m 1b "What about you, [player]?"
+    m 1a "Can you drive at all?"
+    menu:
+        "Yes.":
+            m 1j "Oh, really?"
+            m "That's great!"
+            m 1k "Gosh, you're amazing, you know that?"
+            m "I can just think of the things we could do together, ehehe~"
+            m 1e "Driving can also be dangerout too though."
```

Comment:
correction " Just be careful out there cuz accidents do happen but i guess you already know that"